### PR TITLE
Update the scripts to install ES, Kibana and Marvel from 5 series

### PR DESCRIPTION
**Pull request by karolz-ms**

### CODE REVIEW

File: ES-MultiNode/es-ubuntu-install.sh [hunk 1]
```diff
@@ -240,21 +239,35 @@ es_heap_size=$(free -m |grep Mem | awk '{if ($2/2 >31744)  print 31744;else prin
...
-printf "\nelasticsearch - memlock unlimited" | sudo tee -a /etc/security/limits.conf > /dev/null
+printf "\nelasticsearch  - memlock unlimited" | sudo tee -a /etc/security/limits.conf > /dev/null
+sudo mkdir -p /etc/systemd/system/elasticsearch.service.d
+printf "\n[Service]\nLimitMEMLOCK=infinity\n" | sudo tee -a /etc/systemd/system/elasticsearch.service.d/elasticsearch.conf > /dev/null
 
-
-echo "#################### Starting services ####################"
+echo "#################### Start Elasticsearch service ####################"
+sudo systemctl daemon-reload
 sudo systemctl start elasticsearch.service
```
Comment (nizarq) [inline ES-MultiNode/es-ubuntu-install.sh]: You are starting this on line 269. Is that enough? Should we remove these few lines from here?

File: ES-MultiNode/es-ubuntu-install.sh [hunk 2]
```diff
@@ -240,21 +239,35 @@ es_heap_size=$(free -m |grep Mem | awk '{if ($2/2 >31744)  print 31744;else prin
...
+printf "\nxpack.security.enabled: false\n" | sudo tee -a /etc/elasticsearch/elasticsearch.yml | sudo tee -a /opt/kibana/config/kibana.yml > /dev/null
+printf "xpack.graph.enabled: false\n" | sudo tee -a /etc/elasticsearch/elasticsearch.yml | sudo tee -a /opt/kibana/config/kibana.yml > /dev/null
+printf "xpack.watcher.enabled: false\n" | sudo tee -a /etc/elasticsearch/elasticsearch.yml > /dev/null
+printf "xpack.reporting.enabled: false\n" | sudo tee -a /opt/kibana/config/kibana.yml > /dev/null
 
-echo "#################### Installing ES head plugin ####################"
-sudo /usr/share/elasticsearch/bin/plugin install mobz/elasticsearch-head
+echo "#################### Restarting Elasticsearch and starting Kibana ####################"
+sudo systemctl start elasticsearch.service
+wait_for_elastic_svc;
```
Comment (nizarq) [inline ES-MultiNode/es-ubuntu-install.sh]: what if start fails?

File: ES-MultiNode/es-ubuntu-install.sh [hunk 3]
```diff
@@ -240,21 +239,35 @@ es_heap_size=$(free -m |grep Mem | awk '{if ($2/2 >31744)  print 31744;else prin
...
+sudo systemctl start kibana5.service
```
Comment (nizarq) [inline ES-MultiNode/es-ubuntu-install.sh]: same here? Do we need to check and inform that starting the service worked?

File: ES-MultiNode/es-ubuntu-install.sh (hunk 1 above)
Comment (karolz-ms) [inline ES-MultiNode/es-ubuntu-install.sh]: Good question. Initially I thought installing x-pack requires ES service to be running, but now I doubt it. I am going to try to start ES and Kibana just once and see if it is sufficient

File: ES-MultiNode/es-ubuntu-install.sh (hunk 2 above)
Comment (karolz-ms) [inline ES-MultiNode/es-ubuntu-install.sh]: See previous comment--we'll start ES just once and handle the failure

File: ES-MultiNode/es-ubuntu-install.sh (hunk 3 above)
Comment (karolz-ms) [inline ES-MultiNode/es-ubuntu-install.sh]: I will see if I can add some failure-checking here in a future update

**Pull request merged**